ECMA2Yaml/IntellisenseFileGen/Program.cs

```diff
@@ -544,7 +636,7 @@ public static bool SpecialProcessText(XText xText)
                 }
 
                 // [ISymUnmanagedWriter Interface](~/docs/framework/unmanaged-api/diagnostics/isymunmanagedwriter-interface.md) => ISymUnmanagedWriter Interface
-                pattern = "(\\[(.*?)\\]\\(.*?\\))";
+                pattern = "(\\[(.*?)\\]\\(.*\\))";
```

Comment:
for every regex we use, we should make it a static pre-compiled object that can be reused:

`public static Regex regex = new Regex("pattern", RegexOptions.Compiled);`